scripts/deploy.sh

```diff
@@ -7,17 +7,88 @@ pull_latest() {
 
 rebuild_regs() {
   # Build WCA regulations
-  # Use only one core because we get better backtraces without multiprocessing,
-  # and because we might be on the production server and should not eat up all
-  # cores just building the regulations.
-  regulations/make.py --num-workers 1 --wca
-  regs_folder=WcaOnRails/app/views/regulations
+  # Uses wrc, see here: https://github.com/cubing/wca-regulations-compiler
+  # pdf generations relies on wkhtmltopdf (with patched qt), which should be in $PATH
+  build_folder=regulations/build
+  regs_folder_root=WcaOnRails/app/views
+  regs_folder=$regs_folder_root/regulations
+  regs_version=$regs_folder/version
+  translations_version=$regs_folder/translations/version
+
+  if [ -a $build_folder ]; then
+    rm -rf $build_folder
```

Comment:
Is there any value in checking beforehand? You do a `rm -rf` anyways. I'd just nuke the if statement.